rootfs/scheduler/__init__.py

```diff
@@ -530,23 +529,21 @@ def run(self, namespace, name, image, entrypoint, command, **kwargs):
             name, image, entrypoint, command)
         )
 
-        imgurl = self.registry + '/' + image
         POD = POD_TEMPLATE
-
         l = {
             'id': name,
             'version': self.apiversion,
-            'image': imgurl,
+            'image': image,
             'image_pull_policy': settings.DOCKER_BUILDER_IMAGE_PULL_POLICY,
             'storagetype': os.getenv("APP_STORAGE"),
             'terminationGracePeriodSeconds': settings.KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS  # noqa
         }
 
         if entrypoint == '/runner/init':
             POD = POD_BTEMPLATE
-            l["image"] = image
+            l["slug_url"] = image
```

Comment:
we need to remove the registry part for the the image or the slugrunner will fail with invalid path as this is basically the path of the slug to be downloaded.